OOPs2/Questions.java

```diff
@@ -0,0 +1,108 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Questions implements Comparable<Questions> {
```

Comment:
As this class design to hold one question each, so the name of class can be Question